crates/storage/src/lib.rs

```diff
@@ -201,6 +234,7 @@ mod tests {
         let store = Store::new("test", EngineType::InMemory).unwrap();
         test_store_account(store.clone());
         test_store_block(store.clone());
+        test_store_block_number(store.clone());
```

Comment:
We could have a `test_store_suite` that performs all these tests. That way we'll have only one place to add new tests.
(Unimplemented functionality for rocksdb and sled shouldn't be a problem as they are behind feature flags)

Reply:
Good idea 👍